modules/Microsoft.Web/sites/slots/.parameters/fa.min.parameters.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```

Comment:
I guess  these parameter files are leftovers?

Reply:
yes.. I'll remove the folder `.parameters`

Reply:
Thanks :)